metrics/cli/test/flutter/adapter/flutter_cli_service_adapter_test.dart

```diff
@@ -0,0 +1,41 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/flutter/adapter/flutter_cli_service_adapter.dart';
+import 'package:cli/flutter/cli/flutter_cli.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../test_utils/matchers.dart';
+
+void main() {
+  group("FlutterCliServiceAdapter", () {
+    final flutterCliMock = _FlutterCliMock();
```

Comment:
```suggestion
    final flutterCli = _FlutterCliMock();
```
Also, should we reset this mock after each test?